examples/blog.rb

```diff
@@ -128,7 +128,7 @@ class Style < R '/styles\.css'
       STYLE = File.read(__FILE__).gsub(/.*__END__/m, '')
 
       def get
-        @headers['Content-Type'] = 'text/css; charset=utf-8'
+        @headers['content-type'] = 'text/css; charset=utf-8'
```

Comment:
Will this work not capitalized?

Reply:
Rack 3.0  requires header keys to be uncapitalized.

Reply:
Thank you, I wasn't aware of that!